components/bank/BankBad.vue

```diff
@@ -13,47 +14,37 @@
                 <div class="flex justify-center md:block mb-8 w-full">
                 </div>
                 <div class="font-semibold text-gray-800 text-2xl md:text-4xl tracking-wider mb-2 md:mb-6">
-                    <div v-if="header" v-html="header"></div>
-                    <PrismicRichText v-else :field="bankPage?.data.headline" />
+                    <PrismicRichText :field="bankPage?.data.headline" />
                 </div>
                 <div class="prose sm:prose-lg xl:prose-xl prose-blurb">
-                    <div v-if="summary" v-html="summary"></div>
-                    <div v-else>Your bank doesn't top the charts, but it’s still using your money to lend to fossil fuel
-                        companies and projects that are rapidly accelerating the climate crisis.
-                    </div>
-                </div>  
+                    <PrismicRichText :field="bankPage?.data.description1" />
+                </div>
             </div>
         </template>
 
         <template #section2>
             <PrismicRichText v-if="bankPage?.data?.description2 && bankPage.data.description2.length > 0"
                 class="text-lg md:text-2xl whitespace-pre-line text-gray-900" :field="bankPage.data.description2" />
-            <div v-else-if="details" class="text-lg md:text-2xl whitespace-pre-line text-gray-900" v-html="details"></div>
             <p v-else class="text-lg md:text-2xl whitespace-pre-line text-gray-900" v-text="piggyText"></p>
         </template>
     </BankLayoutBadWorst>
 </template>
 
 
-<script setup lang="ts">import { PrismicDocument } from '@prismicio/types';
+<script setup lang="ts">
+import { PrismicDocument } from '@prismicio/types';
 
 const props = defineProps<{
     name: string,
     website: string,
-    subtitle: string,
-    header: string,
-    summary: string,
-    details: string
     inheritBrandRating: {
         name: string,
         tag: string
     },
-    amountFinancedSince2016: string,
```

Comment:
I think that this is still used to fill the "large amounts" placeholder text?

Reply:
The field remains updated anyway. I think that that should remain in the `piggyText` for now